test/powershell/Modules/Microsoft.PowerShell.Core/Job.Tests.ps1

```diff
@@ -166,3 +166,59 @@ Describe "Debug-job test" -tag "Feature" {
         $result.Command | Should be "<ScriptBlock>"
     }
 }
+
+Describe "Ampersand background test" -tag "CI" {
+    Context "Simple background job" {
+        AfterEach {
+            Get-Job | Remove-Job -force
+        }
+        It "Background with & produces a job object" {
+            $j = write-output Hi &
+            $j | Should BeOfType System.Management.Automation.Job
+        }
+    }
+    Context "Variable tests" {
+        AfterEach {
+            Get-Job | Remove-Job -force
+        }
+        It "doesn't cause error when variable is missing" {
+            Remove-Item variable:name -erroraction Ignore
+            $j = write-output "Hi $name" &
+            rcjb $j -wait | Should Match "Hi "
+        }
+        It "Copies variables to the child process" {
+            $n1 = "Bob"
+            $n2 = "Mary"
+            ${n 3} = "Bill"
+            $j = write-output "Hi $n1! Hi ${n2}! Hi ${n 3}!" &
+            rcjb $j -wait | Should Match "Hi Bob! Hi Mary! Hi Bill!"
+        }
+        It 'Make sure that $PID from the parent process does not overwrite $PID in the child process' {
+            $j = write-output $pid &
+            $cpid = rcjb $j -wait
+            $pid | Should Not BeExactly $cpid
+        }
+        It 'Make sure that $global:PID from the parent process does not overwrite $global:PID in the child process' {
+            $j = write-output $global:pid &
+            $cpid = rcjb $j -wait
+            $pid | Should Not BeExactly $cpid
+        }
+        It "starts in the current directory" {
+            $j = Get-Location | foreach Path &
+            rcjb -wait $j | Should Be ($pwd.Path)
+        }
+    }
+    Context "Backgrounding expressions" {
+        AfterEach {
+            Get-Job | Remove-Job -force
+        }
+        It "handles backgrounding expressions" {
+            $j = 2+3 &
+            rcjb $j -wait | Should Be 5
+        }
+        It "handles backgrounding mixed expressions" {
+            $j = 1..10 | foreach { $_ } &
+            rcjb -wait $j | foreach {$s=0} {$s += $_} {$s} | should be 55
```

Comment:
Please replace alias and add parameter names.

Also please correct register `Should Be` here and above `-Wait`, `Write-Output`, titles and so on.